.markdownlint.json

```diff
@@ -0,0 +1,12 @@
+{
+  "default": true,
+  "MD004": false,
```

Comment:
Why not just enable this and change all the `*` to `-`, it's a pretty trivial fix. Would drive consistency for unordered lists.